### DISCUSSION TITLE
Mqtt and home assistant

### DISCUSSION THREAD

**Discussion by brettus90** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#675] [Q&A]
I have a WiFi module working fine 

only one quirk I have noticed and unaware if it’s meant too do this or not 

Home assistant shows the device as offline in time with the Telemetry Interval in the mqtt settings 

takes a good few seconds to come
Back where it shows as online again 

this has caused some entertainment if it’s offline in those seconds I automate it won’t work 

is this normal or is there a setting I’m missing or do I stretch the telemetry interval bigger smaller etc 

thanks

**Comment by visualapproach** [comment]
Actually, I wouldn't think the interval matters very much, because it will send MQTT updates as soon as anything useful changes. If nothing changes it sends at the interval periods to show "alive". So IIRC the default interval is 60 seconds, and timeout 70 s. Is there a lag between switching something in HA until the spa reacts?

**Reply by brettus90** [reply]
No there is basically no lag 

it’s just the interval seems to cause it to go offline for a short period in HA 

it soon comes back and I have kinda automated round it by checking twice incase it lands right when it’s offline

**Reply by visualapproach** [reply]
okay, but it shouldn't behave like that. What if you set the interval to 15s?

**Reply by brettus90** [reply]
The interval out of the box is 10 minutes ?

**Reply by visualapproach** [reply]
ok, try 60s or less and see if it makes any difference.

**Reply by brettus90** [reply]
No difference 

pic attached if it works .

![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/156433933/868176c4-e918-473e-a02d-454a525cf1a7)

**Reply by visualapproach** [reply]
Strange. Maybe something with the network? Or maybe something wrong with the config in HA. You may loose the history but you could try removing the device in HA and then restart the ESP. It will resend the config after a few minutes.

**Reply by brettus90** [reply]
It’s in honesty not a massive issue 

other things in HA that are not reliant on MQTT also seem to periodically go offline 

but they both use WiFi

so I have this feeling it is possible my EERO APs doing something odd 

That or HA
